### PR TITLE
Dataset upload feature

### PR DESCRIPTION
**Pull request by nandeeshp24**

### CODE REVIEW

**Comment by nikit91** [comment]
@maqboolkhan please take care of the following points:

- [ ] Codacy issues
- [ ] Allow whitespaces and all characters in the "description" field for metadata
- [ ] Test the application behavior upon uploading datasets from kaggle

File: backend-server/src/main/java/org/dice/ida/util/RDFUtil.java [hunk 1]
```diff
@@ -320,20 +329,39 @@ public String getVizIntent(String viz) {
...
 		if (conn != null) {
 			conn.close();
 		}
 		model = null;
 		return attributeOptionalMap;
 	}
+
+	public String addDatasetName(String dsName) {
+		String queryString = "PREFIX ab: <https://www.upb.de/ida/datasets/>" +
+				" INSERT DATA {" +
```
Comment (nikit91) [inline backend-server/src/main/java/org/dice/ida/util/RDFUtil.java:351]: Prone to injections, please look for a better way to do this.

**Review by nikit91 — CHANGES_REQUESTED** [review]
the code looks good to me. However, there are parts that are prone to exploitation through query injections, we should take care of that.

File: backend-server/src/main/java/org/dice/ida/util/RDFUtil.java (hunk 1 above)
Comment (nandeeshp24) [reply backend-server/src/main/java/org/dice/ida/util/RDFUtil.java:350]: taken care of

**Review by nikit91 — APPROVED** [review]
lgtm

**Pull request merged**